Index: thunar-svn-plugin-0.0.3/configure.ac
===================================================================
--- thunar-svn-plugin-0.0.3.orig/configure.ac
+++ thunar-svn-plugin-0.0.3/configure.ac
@@ -27,7 +27,7 @@ dnl *** Initialize autoconf ***
 dnl ***************************
 AC_COPYRIGHT([Copyright (c) 2006-2007 Peter de Ridder <peter@xfce.org>.])
 AC_INIT([thunar-svn-plugin], [tsp_version], [thunar-dev@xfce.org])
-AC_PREFIX_DEFAULT([$(pkg-config --variable prefix thunarx-1 || echo /usr/local)])
+AC_PREFIX_DEFAULT([$(pkg-config --variable prefix thunarx-2 || echo /usr/local)])
 AC_PREREQ([2.50])
 AC_CANONICAL_TARGET()
 
@@ -67,7 +67,7 @@ XDT_I18N([ar cs da en_GB es fr gl hu id
 dnl ***********************************
 dnl *** Check for required packages ***
 dnl ***********************************
-XDT_CHECK_PACKAGE([THUNARX], [thunarx-1], [0.4.0])
+XDT_CHECK_PACKAGE([THUNARX], [thunarx-2], [0.4.0])
 XDT_CHECK_PACKAGE([THUNAR_VFS], [thunar-vfs-1], [0.4.0])
 XDT_CHECK_PACKAGE([APR], [apr-1], [0.9.7])
 XDT_CHECK_PACKAGE([GTK], [gtk+-2.0], [2.6.0])
Index: thunar-svn-plugin-0.0.3/thunar-svn-plugin/Makefile.am
===================================================================
--- thunar-svn-plugin-0.0.3.orig/thunar-svn-plugin/Makefile.am
+++ thunar-svn-plugin-0.0.3/thunar-svn-plugin/Makefile.am
@@ -8,7 +8,7 @@ INCLUDES =								\
 	-DPACKAGE_LOCALE_DIR=\"$(localedir)\"				\
 	$(PLATFORM_CPPFLAGS)
 
-extensionsdir = $(libdir)/thunarx-1
+extensionsdir = $(libdir)/thunarx-2
 extensions_LTLIBRARIES =						\
 	thunar-svn-plugin.la
 
